Dockerfile

```
@@ -21,6 +21,7 @@ ENV LC_ALL C.UTF-8
 # need rsync for deploy script and texlive for building docs
 RUN apt-get install -y --no-install-recommends \
     imagemagick \
+    gsfonts-x11 \
```

I don't think we need the x11 stuff. I tested with this change and it worked.
```suggestion
    gsfonts \
```

---

Makes sense.  Thanks for looking a little deeper.  Updated.